src/include/platform/OTARequestorInterface.h

```diff
@@ -64,6 +64,13 @@ class OTARequestorInterface
     // Send ApplyImage command
     virtual void ApplyUpdate() = 0;
 
+    // Get image update progress in percents unit
+    virtual CHIP_ERROR GetUpdateProgress(uint8_t endpointId, uint8_t & progress) = 0;
```

Comment:
Use `EndpointId` type or just return value from the first endpoint which has OTA Requestor cluster enabled.